### DISCUSSION TITLE
Add "distribution" information for direct file links

### DISCUSSION THREAD

**Discussion by KellyStathis** [datacite/datacite-suggestions#42] [Metadata Schema Suggestions]
### What is the problem that your suggestion solves?

It would be usefull to be able to provide a distribution bloc of information as the one allowed in the dataset schema.org (ex : `{"@type":"DataDownload","encodingFormat":"text/csv","fileSize":42114,"contentUrl":"https://www.seanoe.org/data/00851/96324/data/104641.csv")`

### What solution might meet your needs?

It would allows service that harvest some metadata from datacite end-points to be able to provide a direct link to the data files. 

### What alternatives have you tried or considered?

[Repository] has its own OAI-PMH endpoint that offers to harvest the metadata with the pansimple schema. This schema provides the distribution info. It would be great if we could let all the services that want to harvest [Repository] to use the datacite OAI-PMH endpoint instead.

### Is there anything else you would like to share?

*Suggestion was previously submitted by a user. For DataCite staff reference: https://docs.google.com/spreadsheets/d/1ItM7bYocla83TCYmHI9oZ9Kgt7X0Lznfv6Zls7Gmi88/edit?resourcekey=&gid=1473745305#gid=1473745305*

### What group(s) would benefit from your suggestion?

- [ ] DataCite members registering DOIs and metadata and administering accounts
- [ ] Developers building integrations with DataCite services
- [ ] Harvesters who index DataCite DOIs and metadata
- [ ] Individuals or institutions analyzing DataCite DOIs and metadata
- [ ] DataCite staff
- [ ] Other

**Comment by tedhabermann** [comment]
Lots of discussion of this in the past... See https://wiki.esipfed.org/Distribution_Information_Documentation or https://wiki.esipfed.org/ISO_Distribution_Information for some ideas.

**Comment by paulmillar** [comment]
I can contribute some thoughts on this (or, perhaps, a closely related) topic, collected in [this document](https://docs.google.com/document/d/1waQ5VpEPE_JgH1A7AmPd5pbHyQ6JGyojhEbSZXTJhaM/edit?usp=sharing).

**Comment by KellyStathis** [comment]
We’re looking to gather more info on this topic! To help us move forward, we are especially interested in hearing use cases from metadata harvesters. 

If you’re interested in seeing Content URLs/a Distribution property in the DataCite Metadata Schema, please let us know your thoughts via this form: https://form.typeform.com/to/m5NhS6uo. Let us know if you have any questions—feel free to add to the discussion below.

**Comment by KellyStathis** [comment]
We've shared a new proposal for a "Distribution" property here: https://github.com/datacite/datacite-suggestions/discussions/211
